### ISSUE TITLE
Move from `fs` (pyfilesystem) to `fsspec`

### ISSUE THREAD

**Issue by michaelweinold** [brightway-lca/bw_processing#65]
As suggested by @cmutel, moving to `fsspec` would solve two issues with [`fs` (pyfilesystem)](https://github.com/PyFilesystem/pyfilesystem2):

1. The package is currently not under active development
2. [Code of the sort `if not _WINDOWS_PLATFORM: import grp`](https://github.com/PyFilesystem/pyfilesystem2/blob/8ed9dc495d8ba2f83fbb2a1145d34d92e13644be/fs/osfs.py#L225) causes an error in WASM (Pyodide/JupyterLite-XEUS):

 - https://github.com/brightway-lca/brightway-live/issues/38
 - https://github.com/brightway-lca/brightway-live/issues/39

Note that this was patched in the current https://live.brightway.dev release:

```
import micropip
await micropip.install(
    'https://files.brightway.dev/fs-2.5.1-py2.py3-none-any.whl'
)
```

Still, it would be great to move to `fsspec`, which is supported by both [Pyodide](https://pyodide.org/en/stable/usage/packages-in-pyodide.html) and [emscipten-forge (via the pure Python channel)](https://beta.mamba.pm/channels/conda-forge/packages/fsspec).

This would close:

 - https://github.com/brightway-lca/brightway-live/issues/53

**Comment by cmutel** [comment]
@michaelweinold Told you this was a dirty hack 😛

**Comment by cmutel** [comment]
Fixed in #66

**Comment by michaelweinold** [comment]
Noice! 🎉 Will remove the custom patch with the next `bw_procesing` release!
